package.json

```diff
@@ -19,7 +19,6 @@
   },
   "author": "Daniel A.C. Martin <npm@daniel-martin.co.uk> (http://daniel-martin.co.uk/)",
   "license": "ISC",
-  "type": "module",
```

Comment:
Not entirely sure why this is appearing in the diff as this isn't in `main`. I think it might be something to do with the accidental commit to main that happened.

Can we just check that it doesn't somehow make it into main when this is merged?

Reply:
<img width="700" alt="image" src="https://github.com/UKHomeOffice/lev-api/assets/120499715/bef0c262-b66d-4411-b9c3-9792e8d3e834">

As can be seen from this screenshot, it does not appear directly on the feature branch